modules/chronograf-server/README.md

```diff
@@ -0,0 +1,103 @@
+# Chronograf Server
```

Comment:
Will it be one server or a cluster of servers (ASG)?

Reply:
One server